### PR TITLE
Ds parameters

### PR DESCRIPTION
**Pull request by panamajake86**
# Description
Added and advanced options button that allows the users to set the parameters that would customize their videos.  Utilized dropdown menus and slider input fields.  

Fixes # 
This feature allows the users to customize their videos.

## Type of change
Added feature

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [X] Complete, tested, ready to review and merge, no unit tests

# How Has This Been Tested?

- [x] Tested locally through my machine

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/CreateVideo.js [hunk 1]
```diff
@@ -119,23 +193,269 @@ const CreateVideo = props => {
...
+                    <option value="TOILET PAPER">TOILET PAPER</option>
+                  </select>
+                </label>
+              </div>
+              <div className="jitter">
+                <label>
+                  Jitter
+                  <div
+                    onMouseEnter={handleJitHover}
+                    onMouseLeave={handleJitHover}
```
Comment (hisnameisjimmy) [inline src/components/CreateVideo.js:295]: Dude this is some wild shit

File: src/components/CreateVideo.js [hunk 2]
```diff
@@ -33,14 +33,33 @@ const CreateVideo = props => {
...
   const [titleLoading, setTitleLoading] = useState(false);
 
+  const [optionsClicked, setOptionsClicked] = useState(false);
+
+  const [jitHover, setJitHover] = useState(false);
+  const [deepHover, setDeepHover] = useState(false);
+  const [truncateHover, setTruncateHover] = useState(false);
+  const [pitchHover, setPitchHover] = useState(false);
+  const [tempoHover, setTempoHover] = useState(false);
+  const [smoothHover, setSmoothHover] = useState(false);
```
Comment (hisnameisjimmy) [inline src/components/CreateVideo.js:43]: This is all kinda wild how you're doing this

File: src/components/CreateVideo.js [hunk 3]
```diff
@@ -119,23 +193,269 @@ const CreateVideo = props => {
...
+                  )}
+                  <input
+                    type="range"
+                    name="tempo_sensitivity"
+                    min={0.05}
+                    max={0.8}
+                    step="0.05"
+                    value={videoParams.tempo_sensitivity}
+                    onChange={handleSliderChange}
+                  />
```
Comment (hisnameisjimmy) [inline src/components/CreateVideo.js:424]: A next step for these is probably making them their own components and passing down things as props.

Comment (panamajake86) [reply src/components/CreateVideo.js:424]: I like this a lot. I may girl the repo and try to get this worked out over the weekend.

File: src/components/CreateVideo.js (hunk 2 above)
Comment (panamajake86) [reply src/components/CreateVideo.js:43]: Looking at all these holds and am realizing that a custom hook could've done the work. I may have to refactor some stuff.

**Pull request merged**